src/Octoshift/GithubApi.cs

```diff
@@ -341,6 +341,34 @@ public virtual async Task<string> GetMigrationFailureReason(string migrationId)
             return (string)data["data"]["node"]["failureReason"];
         }
 
+        public virtual async Task<string> GetMigrationLogUrl(string orgLogin, string repositoryName)
+        {
+            var url = $"{_apiUrl}/graphql";
+
+            var query = "query ($orgLogin: String!, $repositoryName: String!)";
+            var gql = @"
+              organization(login: $orgLogin) {
+                repositoryMigrations(last: 1, repositoryName: $repositoryName) {
+                  nodes {
+                    migrationLogUrl
+                  }
+                }
+              }
+            ";
+
+            var payload = new { query = $"{query} {{ {gql} }}", variables = new { orgLogin, repositoryName } };
+
+            var response = await _retryPolicy.Retry(
+              async () => await _client.PostAsync(url, payload),
+              ex => ex.StatusCode == HttpStatusCode.BadGateway
+            );
```

Comment:
Do we expect this to return BadGateway errors in some circumstances?

Reply:
I copied prior art from other functions in this class, i.e.:

https://github.com/github/gh-gei/blob/969e878dcc0f15a5fd2569cb296ca01f979dc6ef/src/Octoshift/GithubApi.cs#L278-L279

Would you like me to remove exception handling for BadGateway?

Reply:
You got it!  Removed in d574f5872c05c71d022eb5ce395f1a9833f8f115!